src/content/sellers-guide/review-solicitations.mdx

```diff
@@ -5,7 +5,7 @@ order: 1
 ---
 
 
-Potential offers are highly encouraged to become familiar with the OASIS+ solicitations. The complete solicitation or Request for Proposals (RFP) is posted on SAM.gov.
+Potential offers are highly encouraged to become familiar with the OASIS+ solicitations. The complete solicitation or Request for Proposals (RFPs) are posted on SAM.gov.
```

Comment:
I know it's late on a Friday, so maybe better with fresh eyes on Monday. I think this is either a singular request: `Request for Proposal is`, or multiples requests: `Requests for Proposals are`. [FAR uses the plural 'requests for proposals'](https://www.acquisition.gov/far/15.203) when talking about multiple requests. Since this is referencing a single solicitation, it seems like the singular makes sense here: `The complete solicitation or Request for Proposal (RFP) is posted on SAM.gov.`